Configurations.md

```diff
@@ -17,6 +17,34 @@ To enable unstable options, set `unstable_features = true` in `rustfmt.toml` or
 
 Below you find a detailed visual guide on all the supported configuration options of rustfmt:
 
+## `allow_chain_call_overflow`
+
```

Comment:
I considered version gating these changes, but it feels more like something that would be beneficial for folks to be able to enable/disable in the long term (not just between now and the 2.x release).

Does this make sense as a config option? If so, do the name and default value of `false` sound reasonable?

Reply:
I think that this feature is useful enough that we want this as the default behavior without the configuration option. I cannot think of situations where we want to turn this off.

Reply:
That makes sense. As I was thinking about it more today, the only reason I was disabling it in my local env while working on this was to minimize diffs stemming from call chains that _weren't_ wrapped. 

I will remove the config option and make this the default behavior 👍